test/test_wiki_trac_rst_convert.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+
+from wiki_trac_rst_convert import convert_content
+
+
+class TracRstToVanillaRst(unittest.TestCase):
+    """
+    Test conversion of content from Trac-flavored reStructuredText to
+    vanilla reStructuredText, that is supported by GitHub
+    """
+
+    def test_empty_adds_newline(self):
+        """Adds a newline at the end of an empty file"""
+        self.assertEqual(convert_content(''), '\n')
+
+    def test_removes_RST_wrapping(self):
+        """Removes the Trac RST markers"""
```

Comment:
I think the docstrings can be better . See https://jml.io/pages/test-docstrings.html 

and https://geck.chevah.com/programming/testing.html#tests-description-docstrings